app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -243,7 +243,7 @@ class ExplorationActivityTest {
 
```

Comment:
nit: line number 58 should be   `@Inject lateinit var context: Context`

Reply:
Done.